### ISSUE TITLE
Access to the probability vector from the pixel-wise view selection inference used in the patch_match_stereo stage

### ISSUE THREAD

**Issue by kudo1026** [colmap/colmap#1218]
**Is your feature request related to a problem? Please describe.**
It may only be me not knowing where to look for it in the current software. But I have an application take could take advantage of the probability vector over the source images produced by the pixel-wise view selection inference in the patch_match_stereo stage. Is there some way I can extract from the software? Or do I have to look for the source code? If so, could you kindly point me roughly where it locates?

**Comment by kudo1026** [comment]
To be more precise, in the [pixel-wise view selection paper](https://www.cs.unc.edu/~ezheng/resources/mvs_2016/eccv2016.pdf) equation (3), the author talked about the vector $P_l(m)$ which is the probability of the patch in source image m being similar to the reference patch (at pixel position $l$). I would like to know if there is any way I can access this probability vector (for length $M$) when the inference is done.

**Comment by ahojnnes** [comment]
Yes, you can save the probability map using https://github.com/colmap/colmap/blob/9f3a75ae9c72188244f2403eb085e51ecf4397a8/src/mvs/patch_match.h#L212

You could just add a new code line here to save it to disk: https://github.com/colmap/colmap/blob/9f3a75ae9c72188244f2403eb085e51ecf4397a8/src/mvs/patch_match.cc#L538
